KPP/fullchem/fullchem_HetStateFuncs.F90

```diff
@@ -92,6 +96,12 @@ SUBROUTINE FullChem_SetStateHet( I,         J,         L,                  &
     RC = GC_SUCCESS
     NA = State_Chm%nAeroType
 
+    ! Define flags for species IDs
+    id_SALAAL = Ind_( 'SALAAL' )
+    id_SALA   = Ind_( 'SALA'   )
+    id_SALCAL = Ind_('SALCAL'  )
+    id_SALC   = Ind_('SALC'    )
+
```

Comment:
I would recommend passing these in as arguments.  Because this routine is called on every (I,J,L) doing a string lookup (even though this the lookup uses a hash table) would probably incur a performance hit.

Reply:
Now fixed in:  23d02914fd787733b6103fe81a34ddd35606bd58